_includes/footer.html

```diff
@@ -31,18 +31,37 @@
       <div class="suggestion-text">{{{text}}}</div>
     </div>
   </div>
-  {% endraw %}
+  {% endraw %}-
   </script>
-
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.4/js/bootstrap.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/1.1.1/anchor.min.js"></script>
-  <script src="{{ site.url }}/js/app/search.js"></script>
+  <script src="https://cdn.jsdelivr.net/g/lodash@3.10.1,hogan.js@3.0.2,algoliasearch@3.8.1,autocomplete.js@0.12.0(autocomplete.min.js)" defer></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap.native/2.0.15/bootstrap-native.min.js" defer></script>
+  <script src="/js/app/anchor.min.js" defer></script>
+  <script src="/js/app/search.js" defer></script>
+  {% if page.homepage %}
+  <script src="/js/app/image-lazy-loader.js" defer></script>
+  {% endif %}
   <script>
-    $(document).ready(function() {
+    document.addEventListener('DOMContentLoaded', function() {
+
+      if ("serviceWorker" in navigator) {
+        navigator.serviceWorker.register("/sw.js").catch(function(e) {
+          console.error("Error during service worker registration:", e);
+        });
+      } else {
+        console.log("service worker is not supported");
```

Comment:
I think we can skip console logging. I'm not a fan of logging on prod websites. It's OK to log if there's some unexpected behavior, but in case service workers are not there, we should just skip it silently.